include/measurement_kit/report/entry.hpp

```diff
@@ -26,6 +26,13 @@ class Entry : private nlohmann::json {
             return JsonDomainError();
```

Comment:
*Note*: we can avoid the changes in this file by using `nlohmann::json` in the unit test. Perhaps this would be better.